### PR TITLE
Fix displaying Registrator commit when it is installed with Pkg.add

### LINKED ISSUE #118: Fix issues with getting Registrator commit
https://github.com/JuliaComputing/Registrator.jl/commit/171d0722222d9bca6352852c811f8f5074393643#commitcomment-33321721

### PR DESCRIPTION
**Pull request by nkottary**
Fix #118

### CODE REVIEW

File: src/regedit/register.jl [hunk 1]
```diff
@@ -417,6 +417,16 @@ function register(
 
         Pkg.Compress.save(compat_file, compat_data)
 
+        reg_pkgs = filter(x -> x.name == "Registrator", Pkg.Display.status(Pkg.Types.Context()))
```
Comment (nkottary) [inline src/regedit/register.jl]: Is there a better way to do this?

File: src/regedit/register.jl [hunk 2]
```diff
@@ -417,6 +417,18 @@ function register(
 
         Pkg.Compress.save(compat_file, compat_data)
 
+        reg_pkgs = Pkg.Display.status(Pkg.Types.Context(),
+                                      [PackageSpec("Registrator",
+                                                   Base.UUID("4418983a-e44d-11e8-3aec-9789530b3b3e"))])
+        if length(reg_pkgs) == 0
+            reg_commit = "unknown"
+        else
+            reg_commit = reg_pkgs[1].new.hash
```
Comment (fredrikekre) [inline src/regedit/register.jl]: This is not a commit hash, its a tree hash.

File: src/regedit/register.jl [hunk 3]
```diff
@@ -417,6 +417,18 @@ function register(
...
+            if reg_commit === nothing    # Registrator is dev'd
+                reg_commit = LibGit2.head(Pkg.dir("Registrator"))
```
Comment (fredrikekre) [inline src/regedit/register.jl]: `Pkg.dir` should not be used, use `pathof(Registrator)` instead.

File: src/regedit/register.jl (hunk 2 above)
Comment (nkottary) [reply src/regedit/register.jl]: How can I get the commit hash?

Comment (fredrikekre) [reply src/regedit/register.jl]: Don't know, I don't think there is an easy way. Why not just use the version instead when `Registrator` is added?

**Pull request merged**